Validation/MuonGEMDigis/plugins/GEMStripDigiValidation.cc

```diff
@@ -286,7 +294,8 @@ void GEMStripDigiValidation::analyze(const edm::Event& event, const edm::EventSe
         Int_t pid = simhit.particleType();
         Int_t pid_idx = getPidIdx(pid);
 
-        me_occ_pid_[key3]->Fill(pid_idx);
+        me_occ_pid_[key2]->Fill(pid_idx);
+        me_occ_pid_eta_[key3]->Fill(pid_idx);
```

Comment:
is the same int quantity filled twice?

Reply:
Yes. But they are filled in different histograms.

Reply:
Yes.
That quantity will be filled twice in different histogram.
But... the key for the second histogram is wrong.
Let me check the histograms. And I will push it again.